src/components/accordion/index.md.njk

```diff
@@ -17,83 +17,123 @@ The accordion component lets users show and hide sections of related content on
+
+### Change how an accordion starts
+
+Users might need some sections to be open from the start. They might also need sections they opened to stay open if they leave and then return to the page.
+
+The accordion component will remember if sections were opened or closed if the user leaves and then returns to the page. You can configure sections to start and stay open, but not closed.
 
 ### Do not disable sections
 
 Disabling controls is normally confusing for users. If there is no content for a section, either remove the section or, if this would be confusing for your users, explain why there is no content when the section is opened.
 
 ## Research on this component
 
-This component is experimental because there are different approaches to accordions in services. More research is needed to determine the best approach, or to know when a different approach works better.
+We updated this component in November 2021 to solve an accessibility issue where the buttons and section labels might be mistaken for links.
 
-The design for this component was originally created and tested by a team at the Government Digital Service (GDS) on the [GOV.UK Service Manual](https://www.gov.uk/service-manual) and [GOV.UK topic pages](https://designnotes.blog.gov.uk/2017/06/29/designing-new-navigation-elements-for-gov-uk/).
+[Read about the research and development that went into improving the accordion component](https://github.com/alphagov/govuk-design-system-backlog/issues/1).
 
 The team made sure the component is accessible, for example that users can interact with it using just the keyboard.
 
-### Known issues and gaps
+#### Users that navigate using ‘elements lists’
 
-The section headings can be mistaken for links, but are treated as buttons. This fails [WCAG 2.1 success criterion 1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships). We plan to fix this issue by the end of June 2021.
+We need to find out more about users that navigate using ‘elements lists’ of headings, buttons, links and other elements — such as users of speech recognition software and partially-sighted users of screen readers.
 
-The plus and minus icon is on the right side of the component, which means users of screen magnifiers might not see it.
+For these users, it might not be clear enough that the section headings are considered buttons, misleading them to navigate (less efficiently) using the show or hide labels.
 
-The 'Open all' button reads out as 'Open all sections' for screen readers. This is potentially confusing for users as the visual content is different to what screen readers read out.
+While inconvenient at first, it’s likely the user will better understand the button area once they interact with it and see its focus state. [See ‘Testing with Dragon’ in the accessibility clinic summary](https://github.com/alphagov/govuk-frontend/issues/2295#issuecomment-906449543).
```

Comment:
Could reword to get rid of dangling modifier ('inconvenient' associated with 'user'). What about something like "While this [behaviour/experience/other] is inconvenient at first, it's likely..."

Reply:
Thanks! I wrote these in a rush so appreciate your eye on this. How's this look?

```suggestion
While this experience is inconvenient at first, it's likely the user will better understand the button area once they interact with it and see its focus state. [See ‘Testing with Dragon’ in the accessibility clinic summary](https://github.com/alphagov/govuk-frontend/issues/2295#issuecomment-906449543). 
```

Reply:
Nice!